### DISCUSSION TITLE
Function with parameters

### DISCUSSION THREAD

**Discussion by proietti1965** [sstaub/TickTwo#7] [General]
How would you use your TickTwo library to call a function like this

void DoSomething(int a, int b)
{
if (a==1 && b==1)
{
Serial.println("Hello");
}
else
{
Serial.println("Goodbuy");
}
}

void setup()
{
DoSomething(1, 1);
}

**Comment by sstaub** [comment]
It is not possible to call functions with parameters.

**Comment by proietti1965** [comment]
Ok. Pity. Thanks anyway
